### DISCUSSION TITLE
KUUKIYOMI: Consider It! [010079600BF22000] crash

### DISCUSSION THREAD

**Discussion by wmdhs12138** [skyline-emu/skyline#325] [General]
Here is the log:
![emulation.sklog.txt](https://github.com/skyline-emu/skyline/files/7865523/emulation.sklog.txt)

**Comment by PixelyIon** [comment]
It crashes with SVC `0x32` unimplemented, this is likely due to it being a Unity game. It will be implemented at a later date and is not a priority at the moment.
